types/react-navigation/react-navigation-tests.tsx

```diff
@@ -552,32 +557,37 @@ class MyBackButton extends React.Component<BackButtonProps & NavigationInjectedP
 
 // withNavigation returns a component that wraps MyBackButton and passes in the navigation prop.
 // If you have class methods, you should have a way to use them.
-const BackButtonWithNavigation = withNavigation(MyBackButton);
-const BackButtonInstance = <BackButtonWithNavigation
-    title="Back" onRef={ref => {
-        // ref is inferred as MyBackButton | null
-        if (!ref) return;
-        ref.triggerBack();
-    }}
-/>;
+
+// This is not used and is causing an error with the assignment to BackButtonWithNavigation below
```

Comment:
I dont think I understand what this comment means. I would like other reviewers to comment on this before merging. Thanks.

Reply:
Hey Sheetal

The four implementations of this comment all reference an instance of a component instead of a constructor. This then causes the error: 
`
JSX element type <ElementInQuestion> does not have any construct or call signatures
`

Please see [this] for a detailed explanation.

Additionally, according to my VSCode the code that is causing these errors is not actually used anywhere anyway.

[this]: <https://stackoverflow.com/questions/31815633/what-does-the-error-jsx-element-type-does-not-have-any-construct-or-call>

Reply:
I dont like the idea of disabling these even if they arent being used however my knowledge of typescript is super limited for this problem... Any insight would be greatly appreciated!